tests/test_other.py

```diff
@@ -11153,6 +11153,9 @@ def test_unistd_fstatfs(self):
 
   # WASMFS tests
 
+  # TODO: This test will only work with the new file system.
+  # Addresses this issue: https://github.com/emscripten-core/emscripten/issues/4017
+  # Also there are cases where an incorrect error is reported.
   @also_with_wasmfs
```

Comment:
I don't hink you need `@also_with_wasmfs` here because this test always runs with wasmfs.

How about renaming the test to `test_wasmfs_dup` to match the filename?

Reply:
I can remove the decorator. Left it in originally to test the JS FS.